scripts/ci/constraints/ci_branch_constraints.sh

```diff
@@ -22,6 +22,8 @@ if [[ ${GITHUB_REF} == 'refs/heads/master' ]]; then
   echo "::set-output name=branch::constraints-master"
 elif [[ ${GITHUB_REF} == 'refs/heads/v1-10-test' ]]; then
   echo "::set-output name=branch::constraints-1-10"
+elif [[ ${GITHUB_REF} == 'refs/heads/v2-0-test' ]]; then
+  echo "::set-output name=branch::constraints-2-0"
```

Comment:
This should be not needed if we continue rebasing. There should be no separate builds targeting 2-0 branch (I believe) so the constraints should continue targeting master. Splitting will only be needed if we actually start diverging.